examples/seismic/acoustic/acoustic_example.py

```diff
@@ -66,8 +66,10 @@ def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=1000.0,
 if __name__ == "__main__":
     description = ("Example script for a set of acoustic operators.")
     parser = ArgumentParser(description=description)
-    parser.add_argument('-nd', dest='ndim', default=3, type=int,
+    parser.add_argument("-nd", dest="ndim", default=3, type=int,
                         help="Preset to determine the number of dimensions")
+    parser.add_argument("-d", "--shape", default=(51, 51, 51), type=int, nargs="+",
+                        help="Determine the grid shape")
```

Comment:
"Determine" here sounds a bit off to me

Reply:
Set, Define, I chose determine to be consistent with the already existing help in line 70.